debian/marquee/03-nginx-conf

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+apt install nginx
+
+# a2dissite doesn't work if nothing is enabled
+rm -f /etc/nginx/sites-enabled/*
+
+cp -p nginx/conf/http.conf /etc/nginx/conf.d/
+cp -p nginx/conf/whatwg.conf /etc/nginx/
+cp -p nginx/conf/whatwg-headers.conf /etc/nginx/
+
+chown -R deploy:deploy /var/www
+rm -rf /var/www/html
+
+/usr/sbin/nginx -t
```

Comment:
why is the path needed?

Reply:
/usr/sbin/nginx -t

> why is the path needed?

Not needed—I’ll change it to just `nginx`